activejob/lib/active_job/logging.rb

```diff
@@ -87,12 +87,25 @@ def queue_name(event)
         def args_info(job)
           if job.arguments.any?
             ' with arguments: ' +
-              job.arguments.map { |arg| arg.try(:to_global_id).try(:to_s) || arg.inspect }.join(', ')
+              job.arguments.map { |arg| serialize_arg(arg).inspect }.join(', ')
           else
             ''
           end
         end
 
+        def serialize_arg(arg)
```

Comment:
You're right. We also don't need the whitelisting that `Arguments` does, so perhaps the other parts are just incidental duplication.

It would be nice to distance this method be renaming it to something that isn't so close to `serialize`. How about just `format`?